tokens/psp22/5DCPp7odM3MTkQEH4X1gFwvA64nfEpxfmxZtSPjaSW1rPJjM/info.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "baldawifhat",
+  "symbol": "balda",
+  "type": "PSP22",
+  "contractAddress": "5DCPp7odM3MTkQEH4X1gFwvA64nfEpxfmxZtSPjaSW1rPJjM",
+  "decimals": 12,
+  "description": "vibe wif frens onchain, but private. no bald, just hat",
+  "website": "https://x.com/AntoniZolciak",
```

Comment:
Hi! The website field is meant to point to a website you own, representing the party deploying the token. So, unless you are Antoni Żółciak, please try replacing it with something else :) You can also leave it blank.

Reply:
Not funny anymore. Fixed